packages/framework/presence/src/test/presenceManager.spec.ts

```diff
@@ -372,6 +442,192 @@ describe("Presence", () => {
 						});
 					}
 
+					describe("when local client disconnects", () => {
```

Comment:
Let's change this to say "and then local client disconnects" to clear up the confusion that test hierarchy says connected and disconnected.